dictsource/be_rules

```diff
@@ -0,0 +1,110 @@
+// This file is UTF8 encoded
+// Translation rules for Belarusian
+
+// Special characters:
+// A  vowel:                   а е ё і о у ы э я
+// B  soft consonant:          б п м ф в дз ц г к х 
+// C  any consonant:           б в г д ж з й к л м н п р с т ў ф х ц ч ш ь ’
```

Comment:
I would say soft sign and especially apostrophe are not consonants

Reply:
removed both